Content/Projectiles/Melee/FireStarterProj.cs

```diff
@@ -10,9 +10,9 @@ internal class FireStarterProjectile : ModProjectile
 
 	public override void SetDefaults()
 	{
-		Projectile.width = 40;
-		Projectile.height = 76;
-		Projectile.aiStyle = 0;
+		Projectile.width = 60;
+		Projectile.height = 60;
+		Projectile.aiStyle = -1;
```

Comment:
Could you provide some detail on what -1 does?

Reply:
aiStyle of 0 for both NPCs and Projectiles do have some basic functionality - NPCs turn to face the player, and I forget what Projectiles have for it. It's almost unnoticeable, but it's standard to say -1 for "entirely custom behaviour".